.travis.yml

```diff
@@ -31,7 +31,7 @@ install:
   - make golint
   - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.17.4/bin/linux/amd64/kubectl
     && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
-  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.8.1/minikube-linux-amd64
+  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.17.1/minikube-linux-amd64
```

Comment:
Can we run the tests in github actions instead of travis?

Reply:
can we take this in a separate PR?

Reply:
The tests will be migrated to github actions in a separate PR.